core/src/tangram.cpp

```diff
@@ -275,6 +275,7 @@ void Map::resize(int _newWidth, int _newHeight) {
     impl->renderState.viewport(0, 0, _newWidth, _newHeight);
 
     impl->view.setSize(_newWidth, _newHeight);
+    impl->selectionBuffer = std::make_unique<FrameBuffer>(_newWidth/2, _newHeight/2, false);
```

Comment:
Yes, need to check again. Though it seems that is should be available since GLES 2.0  http://aras-p.info/blog/2012/10/17/non-power-of-two-textures/